README.md

```diff
@@ -23,12 +23,12 @@ See [action.yml](action.yml) for more details
 ### Parameters
 
 `version`
-> latest, v4, v4.0, v4.0.2, v3, v3.0, v3.0.10, 2.5-ss, 2.5-sc<br/>
-> See this [Docker Image](https://hub.docker.com/r/jacobalberty/firebird) for available versions.
-  and further details on input environment variables
+> latest, 5, 5.0.2, 5-noble, 5-jammy, 4, 4.0.5, 3, 3.0.9, ...<br/>
```

Comment:
The ordering of tags for 3.0 on https://hub.docker.com/r/firebirdsql/firebird/tags is a bit random, so the latest 3.0 is 3.0.12, but the first you'll find in the list is 3.0.9 when sorted by "newest".

Reply:
This message is just a suggestion.
But definitively, the ordering had beaten me.
It would be interesting to have other tags like:
 - 3-latest
 - 4-latest
 - 5-latest

Reply:
Effectively, that are the `3`, `4` and `5` tags.